activerecord/lib/active_record/encryption/configurable.rb

```diff
@@ -52,7 +52,8 @@ def encrypted_attribute_was_declared(klass, name) # :nodoc:
 
         def install_auto_filtered_parameters_hook(application) # :nodoc:
           ActiveRecord::Encryption.on_encrypted_attribute_declared do |klass, encrypted_attribute_name|
-            application.config.filter_parameters << encrypted_attribute_name unless ActiveRecord::Encryption.config.excluded_from_filter_parameters.include?(encrypted_attribute_name)
+            filter_parameter = [("#{klass.model_name.element}." if klass.name), encrypted_attribute_name.to_s].compact.join
```

Comment:
It's a nitpick really but I feel like the "dot" is a little bit hidden so perhaps we could make it a little explicit, like:
```suggestion
            filter_parameter = [("#{klass.model_name.element}" if klass.name), encrypted_attribute_name.to_s].compact.join(".")
```

Reply:
Thanks @nvasilevski I grabbed that :) 👍